docs/core/AFRAME.md

```diff
@@ -1,12 +1,20 @@
 ---
-title: AFRAME
+title: AFRAME Globals
 type: core
 layout: docs
 parent_section: core
 order: 10
 ---
 
-A-Frame exposes its public interface through the `window.AFRAME` browser global. This same interface is exposed if requiring with NPM (`require('aframe');`). `AFRAME` can be used to register new things and extend AFRAME's capabilities.
+A-Frame exposes its public interface through the `window.AFRAME` browser
+global. This same interface is exposed if requiring with NPM
```

Comment:
well, technically it's requiring a Node package installed from npm. and notice the casing.